service/src/main/java/org/cbioportal/genome_nexus/service/mixin/TranscriptConsequenceMixin.java

```diff
@@ -71,6 +72,9 @@ public class TranscriptConsequenceMixin
     @JsonProperty(value="consequence_terms", required = true)
     private List<String> consequenceTerms;
 
+    @JsonProperty(value = "alpha_missense", required = true)
+    private AlphaMissense alphaMissense;
+
```

Comment:
Is this used anywhere? Seems like the only `AlphaMissenseMixin` is needed , right?

Reply:
solved it